src/System.Device.Gpio/Interop/Unix/Libc/Interop.threads.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Disable these StyleCop rules for this file, as we are using native names here.
+#pragma warning disable SA1300 // Element should begin with upper-case letter
+#pragma warning disable SA1307 // Field should begin with upper-case letter
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal static bool SetCurrentThreadHighPriority()
+    {
+        IntPtr thread = pthread_self();
+        sched_param sched = new sched_param();
+
+        sched.sched_priority = sched_get_priority_min(SCHED_FIFO);
+        return pthread_setschedparam(thread, SCHED_FIFO, ref sched) == 0;
+    }
+
+    internal static void SetCurrentThreadNormalHighPriority()
+    {
+        nice(19);
```

Comment:
Isn't this the lowest priority? Thread is nice when it shares more time :smile:

Reply:
Yes, I know. I just copied this file from devices/Interop and left as-is. The method is currently not used here (and I'm confused about it's name as well, but I didn't check where it's used now)